packages/edit-widgets/src/components/widget-area/index.js

```diff
@@ -22,7 +23,9 @@ function WidgetArea( { title, initialOpen } ) {
 					onInput={ updateBlocks }
 					onChange={ updateBlocks }
 				>
-					<BlockList />
+					<ObserveTyping>
```

Comment:
- What about `WritingFlow`?
 - This makes me wonder if `WritingFlow` and `ObserveTyping` could be included by default in `BlockEditorProvider`?
 - I think at least `ObserveTyping` could be part of `WritingFlow`, Thoughts? @aduth @ellatrix

Reply:
>This makes me wonder if WritingFlow and ObserveTyping could be included by default in BlockEditorProvider?

My problem with it would be more conceptual, i.e. defining what it is that the provider is meant to be doing.

My understanding was that it’s just preparing the state context.

The point of making these things individual components was as an opt-in thing

I’m not _opposed_ to baking some of it into the default behavior, but it does muddy the definition of (a) what the provider is meant to be doing and (b) what baseline to we judge against to decide when and when not to isolate behavioral components

Reply:
It looks like we went in the direction of removing functionality from BlockEditorProvider e.g: SlotFillProvider was removed.
I included the writing flow addition. 
 guess we can keep this approach where we have granular components, and then after we can try to expose a more generic version that includes a block editor with the most common functionality e.g: BlockEditor with SlotFillProvider, ObserveTyping, WrittingFlow etc...
The widget screen would not use this version because it is a complex scenario with multiple editors on the same page and we need some low-level control.

I'm also opened to merging ObserveTyping inside WrittingFlow. @ellatrix would that be a positive improvement? Either way, I guess we can make that change in other PR if this one gets approved first.